app/channels/application_cable/channel.rb

```diff
@@ -1,3 +1,5 @@
+# frozen_string_literal: true
```

Comment:
Do you tell me about what is the mean of this code

Reply:
Dạ theo em tìm hiểu thì đoạn code đó có nghĩa là sẽ thiêt lập cố định các chuỗi trong file đó.